### PR TITLE
Report bug, issue template

### PR DESCRIPTION
**Pull request by blukai**
image

@howardchung Maybe u can supplement issue template and fix mistakes if exists 😆

### CODE REVIEW

**Comment by howardchung** [comment]
This is very cool, I didn't know about GitHub issue templates.  I'll update the text a bit later.

Does the report bug link work well on mobile (with the nav drawer?)

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -36,20 +37,31 @@ const navbarPages = [
...
 const LinkGroup = () => (
   <ToolbarGroup className={styles.verticalAlign}>
     {navbarPages.map(page => (
       <div key={page.name} className={styles.tabContainer}>
         {page.external ?
-          <a href={page.path} className={styles.tab} rel="noopener noreferrer" target="_blank">{page.name}</a> :
-          <Link to={page.path} className={styles.tab}>{page.name}</Link>}
+          <a href={page.path} className={styles.tab} rel="noopener noreferrer" target="_blank">{page.name}</a>
+          : <Link to={page.path} className={styles.tab}>{page.name}</Link>
+        }
```
Comment (dartharaknus) [inline src/components/Header/Header.jsx]: Why the change in style?

**Comment by dartharaknus** [comment]
Checked every resolution and looks good.

File: src/components/Header/Header.jsx (hunk 1 above)
Comment (howardchung) [inline src/components/Header/Header.jsx]: I think one of the linting rules changed or something

**Comment by blukai** [comment]
> Does the report bug link work well on mobile (with the nav drawer?)

Yes 🦄
